test/e2e-common.sh

```diff
@@ -143,12 +143,21 @@ function install_head() {
   kubectl apply -f "${EVENTING_KAFKA_CHANNEL_ARTIFACT}" || return $?
 }
 
+function set_default_channel(){
+  kubectl patch configmap default-ch-webhook \
+  -n knative-eventing \
+  --type merge \
+  -p '{"data":{"default-ch-config": "clusterDefault:\n  apiVersion: messaging.knative.dev/v1beta1\n  kind: KafkaChannel"}}'
+}
+
 function test_setup() {
 
   build_components_from_source || return $?
 
   install_head || return $?
 
+  set_default_channel  || return $?
+
```

Comment:
There is no need of this, see https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/test/config/default-ch-webhook.yaml

That is applied by `ko apply -f ./test/config/` line 164

Reply:
Hmm, ok, setting that in that shell script was necessary for me locally. I will first check if things pass with this change and then can remove and see what happens.

Reply:
Sure, I think you need to rebase locally to get it on your branch

Reply:
Ah, ok, it is a new change... I was trying last week and my change was required back then.

Reply:
Ok, reverted this change